tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
```

Comment:
`To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.`

- As written this implies "identify" is enough to reach "net zero goals"? Isn't it just one step?
- "We" - tag? users of cloud native technologies? everyone who has a data center? 
- Do we want to "mitigate" as well? in addition to "identify"?

Reply:
I think you have your answer right there, "we" must refer to the TAG, but since it is a global problem, I think it is actually correct to say "we" instead of the "the TAG".

Reply:
@dims  I updated here:  https://github.com/catblade/toc/pull/3

I have not pushed yet, because I'm waiting for the community to respond.

How about the wording:
we need to first identify how to reduce the environmental footprints of increasingly large data centers and then help the community take action to mitigate these footprints.

Reply:
@catblade sounds good!

Reply:
Since this is a charter of the TAG, suggest to change to "The TAG will first ..." unless "we" can be clearly defined and people involved in the "we" are committed to the work.

Reply:
Better to define the scope of actions this TAG will cover to mitigate these footprints.

Reply:
@cathyhongzhang  we have a scope section-it has many items defined there.

Reply:
@cathyhongzhang I'm confused on your other comment regarding the "we".  I would assume "we" is the TAG, as it is the TAG charter.  I don't see another real interpretation here.  Am I misunderstanding your comment?